phaser/cli/commands/run.py

```diff
@@ -82,4 +82,7 @@ def execute(self, args):
             self.pipeline.init_source(source, args[source])
 
         print(f"Running pipeline '{self.pipeline.__class__.__name__}'")
-        self.pipeline.run()
+        try:
+            self.pipeline.run()
+        except phaser.DataException as e:
+            print("Error processing data.  ", e.message)
```

Comment:
I believe this is meant to catch the `DataException` raised in `Pipeline#320`, but can other `DataException`s get raised elsewhere that would inappropriately bubble here?

Reply:
Probably, but I couldn't proactively think of things that would likely do so? I'm sure we'll find more errors that we'll want to report better, as we gain more use cases and users